source/reference/mongosync.txt

```diff
@@ -296,6 +296,24 @@ error occurs, the ``mongosync`` log may contain the word "error' but
 ``mongosync`` is still able to complete the sync. In the case that a
 sync does not complete, ``mongosync`` writes a fatal log entry. 
 
+UUIDs
+~~~~~
+
+``mongosync`` creates new universally unique identifiers (UUIDs) for
+collections on the destination cluster. There is no relationship between
+UUIDs on the source cluster and the destination cluster. If applications
+contain hard-coded collection UUIDs (which MongoDB does not recommend),
+``mongosync`` requires additional updating to work with a migrated
+cluster. 
+
+Sorting
+~~~~~~~
+
+``mongosync`` creates a new natural (without any sort method specified)
+order on the destination cluster. If applications depend on document
+order without a defined sort method, they may require additional updating to
+work with a migrated cluster. 
```

Comment:
I think this can be cleaned up a bit and also be more specific about what needs to be updated. Perhaps:

> If applications depend on document order, but don't have a defined sort method, they may need to be updated to specify the expected sort order before they will work properly with the migrated cluster.

Reply:
Done!